.github/workflows/release.yml

```diff
@@ -81,10 +84,21 @@ jobs:
           pipeline: elastic-otel-java-release
           waitFor: true
           printBuildLogs: false
+          artifactName: releases
```

Comment:
[question] does it means that we only have one artifact (jar) file generated for the whole project ?, in practice we have more than one so do we have to do the same for each jar generated by the project ?

Reply:
> [question] does it means that we only have one artifact (jar) file generated for the whole project ?,

Good question, I probably was wrong when I though that the only generated file was the one in https://github.com/elastic/elastic-otel-java/blob/6ed4c262282a1c16ba3991d2d4b13044bcc3259e/.buildkite/release.yml#L11 or files


>  in practice we have more than one so do we have to do the same for each jar generated by the project ?

the provenance accepts glob patterns, so it can be anything. At the moment, I only used `agent/build/libs/elastic-otel-javaagent-*.jar"` but if there are more jar files, please let me know and I can adjust the attestation to be done in all the files.

Reply:
Currently we have 4 of them, and only the "agent" was covered in buildkite, they are already published in the snapshots as we can see here: https://oss.sonatype.org/service/local/repositories/snapshots/content/co/elastic/otel/.

I've opened https://github.com/elastic/elastic-otel-java/pull/224 to fix that. Once this is merged then you should be able to add the others.

Reply:
Awesome, thanks @SylvainJuge

Reply:
#224 has just been merged, you are good to go @v1v .

Reply:
https://github.com/elastic/elastic-otel-java/pull/223/commits/2fa7c2aa17dc934d29bf19728c4d13c8dfd8cd39 is the proposal to search for all `jar` files in the `build` folders, then there is no need to maintain the list of artifacts.